security/security-design/fn-datasafe-dbaudit-to-oci-logging/terraform/functions.tf

```diff
@@ -0,0 +1,55 @@
+###############################################################################
+# Copyright (c) 2022, 2023, Oracle and/or its affiliates.  All rights reserved.
+# This software is dual-licensed to you under the Universal Permissive License
+# (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl.
+###############################################################################
+#
+# Author: Fabrizio Zarri
+#
+################################################################################
+
+
+resource "oci_functions_application" "FunctionApp" {
+  compartment_id = var.compartment_ocid
+  display_name   = local.functionapp_display_name
+  subnet_ids     = [var.create_network ? module.setup-network[0].subnet_ocid : var.subnet_ocid] 
+}
+
+resource "oci_functions_function" "fun1" {
+  depends_on     = [null_resource.FunctionAppPush2OCIR]
+  application_id = oci_functions_application.FunctionApp.id
```

Comment:
Terraform naming convention is all lower case and words separated by '_'.

Reply:
I have update the name of all variables and references. Now are only present lower case and words separated by '_'.